Assets/Scripts/SceneLoader.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.SceneManagement;
+
+public class SceneLoader : MonoBehaviour
+{
+    public GameObject LoadingPanel;
+    public GameObject ButtonPanel;
+    public Slider slider;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown("1"))
```

Comment:
We don't have a keypad on anything other than a PC or Mac - we should get out of this habit and use the input manager to map actions to different platform-specific bindings. This could probably wait til next PR, but it's a good habit to adopt early

Reply:
I always add these in because I like them when testing, but I agree, let's not rely on this pattern since we're multi-platform.  Removed.

Reply:
I also removed any other references that we had added to `Input.GetKeyDown`